sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1600,8 +1601,13 @@ public void close() throws IOException {
      * Writes just the values to Kafka. This is useful for writing collections of values rather
      * thank {@link KV}s.
      */
+    @SuppressWarnings({ "unchecked" })
     public PTransform<PCollection<V>, PDone> values() {
-      return new KafkaValueWrite<>(toBuilder().build());
+      return new KafkaValueWrite<>(
+          toBuilder()
+          .setKeySerializer((Class<? extends Serializer<K>>) StringSerializer.class)
```

Comment:
Does this actually compile for you? It causes 'incompatible types' error for me. But the following compiles : 
`.setKeySerializer((Class) StringSerializer.class)`